lib/spark_api/models/saved_search.rb

```diff
@@ -6,12 +6,25 @@ class SavedSearch < Base
       include Concerns::Savable,
               Concerns::Destroyable
 
+      attr_accessor :newsfeeds
+
       self.element_name="savedsearches"
 
+      def initialize(attributes={})
+        @newsfeeds = nil
+        super(attributes)
+      end
+
       def self.provided()
         Class.new(self).tap do |provided|
           provided.element_name = '/savedsearches'
           provided.prefix = '/provided'
+          def is_provided_search?
+            true
+          end
+          def newsfeeds
+            []
```

Comment:
I was having some problems with the tests and this provided class modifier method. I thought about creating a subclass, now that there are two separate attributes and two new methods. I set the provided newsfeeds to `[]` mainly to get the tests working. :disappointed: